graphql-router/src/main/java/com/networknt/graphql/router/GraphqlEndpointSource.java

```diff
@@ -0,0 +1,28 @@
+package com.networknt.graphql.router;
+
+import com.networknt.graphql.common.GraphqlUtil;
+import com.networknt.handler.config.EndpointSource;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.Arrays;
+
+/**
+ * Lists standard GraphQl endpoints at /graphql and /subscriptions.
+ */
```

Comment:
Would you mind updating this comment since the one bellow it states that `/subscriptions` doesn't need to be listed?

Reply:
Ah, yes. That's from before I realised that the websocket handling bypasses the paths completely.